Tools/WinMLRunner/src/BindingUtilities.h

```diff
@@ -3,69 +3,132 @@
 #include <time.h>
 #include "Common.h"
 #include "ModelBinding.h"
+#include <MemoryBuffer.h>
 #include "Windows.AI.Machinelearning.Native.h"
+#include "DirectXPackedVector.h"
```

Comment:
"DirectXPackedVector.h" already included inside Common.h

Reply:
Since BindingUtilities is a header file and run.cpp already includes BindingUtilities.h as well as (Common.h, OutputHelper.h, ModelBinding.h) and when the compiler goes and compiles run.cpp, these header files are already included. No need to include OutputHelper.h and CommandLineArgs here.

Reply:
Removed.

Reply:
Eventually refactoring will be needed so that implementation is separated from declaration in .cpp and .h files to avoid confusion like this.

Reply:
I think that's a good idea.  Forward declaring templates is actually a great way to do this, and to cut down on unnecessary compilation of templates that are not kept (template functions are actually generated every time the header is included in a cpp file, then the linker discards all but one).  Anyway... a little refactoring may be a very good thing. :)